superset-frontend/src/views/CRUD/data/common.ts

```diff
@@ -36,8 +36,8 @@ export const commonMenuData = {
     {
       name: 'Saved Queries',
       label: t('Saved Queries'),
-      url: '/sqllab/my_queries/',
-      usesRouter: false,
+      url: '/savedqueryview/list/',
+      usesRouter: true,
```

Comment:
Turns out it does not, I think I misunderstood how we were using the feature flags. In which case the new Databases view feature flag also got broken with my last routing update

Reply:
To be clear, routing from the nav bar still works with the feature flag, it's just routing within the `SubMenu` that needs an extra feature flag check

Reply:
Yup, since databases is already done this shouldn't be a problem. I'm planning to remove the `SIP_34_DATABASE_UI ` flag soon. This is mainly to avoid users landing on a blank/under construction page when `ENABLE_REACT_CRUD_VIEWS=True`. If you want to add a feature flag check here you're welcome to, but I'd say for now it's fine if it routes through the backend and we'll switch this on once the saved queries ui is done.